web-ui/src/main/resources/catalog/components/common/map/mapsManager.js

```diff
@@ -197,32 +203,37 @@
               }
             }
 
-            // layers
+            // load layers from Settings
             if (config.layers && config.layers.length) {
               config.layers.forEach(function(layerInfo) {
                 gnMap.createLayerFromProperties(layerInfo, map)
                   .then(function(layer) {
                     if (layer) {
-                      map.addLayer(layer);
+                      layer.displayInLayerManager = false;
+                      layer.set("group", "Background layers");
+                      layer.set("fromGNSettings", true);
+                      gnViewerSettings.bgLayers.push(layer);
                     }
                   });
               });
             }
+            
             if(type == this.VIEWER_MAP) {
               if (mapParams.wmsurl && mapParams.layername) {
                 gnMap.addWmsFromScratch(map, mapParams.wmsurl,
                   mapParams.layername, true).
 
                 then(function(layer) {
                   layer.set('group', mapParams.layergroup);
+                  layer.displayInLayerManager = true;
                   map.addLayer(layer);
                 });
               }
             }
-          }.bind(this));
+          });
 
           // save the promise on the map
-          map.set('creationPromise', creationPromise);
+          map.set('creationPromise', mapReady);
```

Comment:
For the creationPromise we are interested on this promise, not the "creationPromise" which no longer exists, as now we handle the "creationPromise" with the owsContextLoaded event.